tutorials/Linear Combination of Unitaries/LCU Tutorial .ipynb

```diff
@@ -0,0 +1,395 @@
+{
```

Comment:
<p>Please move the implementation of the Within Apply on Toffoli into a 'HINT' comment with: &lt;details&gt; &lt;summary&gt; summary goes here &lt;/summary&gt; &lt;/details&gt;</p><p> </p><p>This is a bit confusing when it is a part of the tutorial itself..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Please note this comment</p>